client/packages/system/src/Patient/api/hooks/usePatientVaccineCard.ts

```diff
@@ -1,4 +1,6 @@
+import { useQuery } from '@openmsupply-client/common';
 import { usePatientGraphQL } from '../usePatientGraphQL';
+import { VACCINATION_CARD } from './keys';
 
 export const usePatientVaccineCard = (
   patientId: string,
```

Comment:
Ah, I see we only need the `programEnrolmentId` -- so we can get rid `patientId` from the input parameters.

(I can do this in my PR if you like?)

Reply:
Ah yep missed that one - eh its a quick change, I'll chuck it in with the other requested changes now